docs/starters.yml

```diff
@@ -1647,3 +1647,15 @@
     - Comes with React Helmet for adding site meta tags
     - Add Disqus
     - Nice Pagination
+- url: https://master.d2f5ek3dnwfe9v.amplifyapp.com/
```

Comment:
Would it be possible to actually create a proper dns name for it so it updates with new versions. This seems like an url that might not be in sync with the git repo. Correct me if i'm wrong

Reply:
Sweet, I was worrying about `d2f5ek3dnwfe9v` inside the url but if that url doesn't change, than 👍